assets/js/src/core/components/toolbar/toolbar.tsx

```diff
@@ -23,20 +24,25 @@ export interface ToolbarProps {
   theme?: 'primary' | 'secondary'
   position?: 'top' | 'bottom'
   size?: 'small' | 'default'
+  padding?: SizeDefinition
+  margin?: SizeDefinition
 }
 
-export const Toolbar = ({ children, size = 'default', justify = 'space-between', align, theme = 'primary', position = 'bottom', ...props }: ToolbarProps): React.JSX.Element => {
+export const Toolbar = ({ children, size = 'default', justify = 'space-between', align, theme = 'primary', position = 'bottom', padding, margin, ...props }: ToolbarProps): React.JSX.Element => {
   const { styles } = useStyles()
   const classes = [
     styles.toolbar,
     'toolbar',
     `toolbar--theme-${theme}`,
     `toolbar--position-${position}`,
     `toolbar--size-${size}`
-  ].join(' ')
+  ]
+
+  const paddingClasses: string[] = getSizingClasses('p', padding)
+  const marginClasses: string[] = getSizingClasses('m', margin)
 
   return (
-    <div className={ classes }>
+    <div className={ `${classes.join(' ')} ${paddingClasses.join(' ')} ${marginClasses.join(' ')}` }>
```

Comment:
Wouldn't the box component also work in this case? 
Since it's designed to manage these paddings and margins?